templates/artist/artist_detail.html

```diff
@@ -104,9 +104,9 @@
             var myChart5 = new Chart(ctx, {
                 type: 'pie',
                 data: {
-                    labels: ["Me", "Other Investors", "Artist"],
+                    labels: ["Me", "Other Investors", "Artist", "Brands"],
```

Comment:
This was meant to be a temporary fix. Right now it says "undefined" when another user is added so I just put it as a placeholder so it doesn't look like an error.